### ISSUE TITLE
Should default gl::clear() color be ColorA::zero() instead of black()?

### ISSUE THREAD

**Issue by richardeakin** [cinder/Cinder#1704]
Currently if you're writing to an FBO with alpha blending enabled, you can accidentally clear out the entire alpha channel with a call to `gl::clear()`. Should we change the default to be `ColorA::zero()` so it writes zeros to the alpha channel too?

Relevant line [here](https://github.com/cinder/Cinder/blob/aeab54cfcae82bbfdb4756c3f2c9d26000c75788/include/cinder/gl/wrapper.h#L91).

**Comment by paulhoux** [comment]
I think that makes perfect sense.

**Comment by mhintz** [comment]
But if you called `gl::clear` and passed an alpha-free `Color`, it would still clear the alpha channel to 1?

**Comment by richardeakin** [comment]
@mhintz yes, since the `ci::Color` would get converted to a `ci::ColorA` when passed in as argument, with an alpha channel of 1. I'd only like to change the default `ColorA` argument [here](https://github.com/cinder/Cinder/blob/f86ddd8d89b58af528f3dd53d455c0aada4cdf06/include/cinder/gl/wrapper.h#L91), so that by default all channels get cleared to 0.

Marking this for 0.9.2 as I think we're too late in the current release cycle to introduce functionality changes.
